### DISCUSSION TITLE
DC Aktualisierung/Abstürze alle 7 Tage

### DISCUSSION THREAD

**Discussion by blackbasket** [OpenCCU/OpenCCU#1207] [🇩🇪 Fragen&Antworten]
Hallo zusammen,

wie erfolgt bei der RM die Aktualisierung des DC?

Hintergrund der Frage:
Meine RM/CCU3 - bzw. genauer das HmIP-Modul - stürzt alle 7 Tage (+/- ein paar Stunden) ab. Ab dem Moment ist keine HmIP-Kommunikation mehr möglich und es werden keine Programme mehr ausgeführt. Eine Steuerung der Rollos mittels CUxD (zB per Home App) ist weiter möglich. Ich habe bereits die Hardware getauscht, bin von CCU auf RM umgestiegen, aber ich finde den Fehler nicht.

Was mir aber seit dem Umstieg von CCU auf RM auffällt, ist, dass der DutyCycle-Wert weiterhin aktualisiert wird. Dies ist mit der Original-FW "logischerweise" nicht passiert, da es sich um ein Programm von mir gehandelt hat.

Ich weiß nicht mehr weiter... Abgesehen von CUxD ist auch nichts mehr zusätzlich installiert. Selbst HAP-Homematic läuft inzwischen parallel auf einem Ubuntu.

Ich bin für jede Idee dankbar...

LG,
Marcel

Edit: Ich habe natürlich nicht auf RM gewechselt, weil ich erwartet habe, dass sich das Problem in Luft auflöst. Aber zumindest auf Hilfe von WatchDog hatte ich gehofft ;) Kann man da eigentlich irgendwas konfigurieren oder existiert der nur und macht, "was er will"?

**Comment by jens-maus** [comment]
> wie erfolgt bei der RM die Aktualisierung des DC?

Die Aktualisierung des DC funktioniert im Grunde genauso wie bei einer CCU3, nämlich im Funkmodul selber findet das statt. RaspberryMatic bzw. die WebUI einer CCU3 stellt dann dieser nur entsprechend dar, mehr nicht.

> Hintergrund der Frage:
> Meine RM/CCU3 - bzw. genauer das HmIP-Modul - stürzt alle 7 Tage (+/- ein paar Stunden) ab. Ab dem Moment ist keine HmIP-Kommunikation mehr möglich und es werden keine Programme mehr ausgeführt. Eine Steuerung der Rollos mittels CUxD (zB per Home App) ist weiter möglich. Ich habe bereits die Hardware getauscht, bin von CCU auf RM umgestiegen, aber ich finde den Fehler nicht.

Was genau verstehst du denn unter "das HMIP-Module stürzt alle 7 Tage ab"? Was genau stürzt denn bitte "ab" und wie nimmst du das wahr? Denn wenn du mittels CUxD ja prinzipiell die Rollos noch steuern kannst funktioniert ja die Kommunikation mit deinem HmIP Geräten noch, oder nicht? Definiere also bitte was du unter "abstürzen" verstehst in diesem Zusammenhang und was genau du meinst.

**Reply by blackbasket** [reply]
Hallo Jens,

erst einmal danke für die Antwort und Frohe Ostern :)

> Die Aktualisierung des DC funktioniert im Grunde genauso wie bei einer CCU3, nämlich im Funkmodul selber findet das statt. RaspberryMatic bzw. die WebUI einer CCU3 stellt dann dieser nur entsprechend dar, mehr nicht.

Und wie wird der Wert der SV DutyCycle aktualisiert? Mich wundert halt, dass dies bei der RM weiterhin funktioniert.

> Was genau verstehst du denn unter "das HMIP-Module stürzt alle 7 Tage ab"? Was genau stürzt denn bitte "ab" und wie nimmst du das wahr?

Nach der Zeit ist keine Kommunikation von HmIP-Komponenten mit der RM mehr möglich. Auf den Heizungsradiatoren blinkt dann auch das Funksymbol. (Andere Geräte mit Display besitze ich nicht.) Die anderen Geräte erhalten gemäß LED einfach keine Antwort mehr.

Dass die CCU keine Programme mehr ausführt, stelle ich u. a. daran fest, dass meine netatmo-Geräte keinen Sync mehr durchführen und die Uptime der CCU in der SV nicht mehr aktualisiert wird.

top gibt mir dann jedes Mal folgenden Prozess mit entsprechender CPU usage an:
`  PID  PPID USER     STAT   VSZ %VSZ %CPU COMMAND
  941     1 root     S    3463m 360%  21% java -Dos.arch=aarch64 -Xmx128m -DlibN`

Darüber hinaus kann ich die 25% CPU usage in PRTG bereits ca. 5-15 min vor dem "Absturz" sehen. Sonst komme ich nur bei den nächtlichen Jobs auf diesen Peak. Gefühlt tritt schon ca. 4-8h vor dem Verhalten eine deutliche Verzögerung bei der Ausführung von Programmen auf.

> Denn wenn du mittels CUxD ja prinzipiell die Rollos noch steuern kannst funktioniert ja die Kommunikation mit deinem HmIP Geräten noch, oder nicht?

Nein, die Rollos sind Somfy RTS über nanoCUL und CUxD.

Ich hoffe, ich konnte es jetzt ein wenig besser beschreiben.

**Reply by blackbasket** [reply]
Hallo @jens-maus ,

hier mal ein kurzes Update. Der Downgrade auf 3.53.34... hat keine Besserung gebracht. Ich habe inzwischen die letzte - sicher funktionierende - Config aus November vom Tape wiederhergestellt.

Dabei ist mir dann aufgefallen, dass die letzte FW ohne Probleme schon 3.53.30... war. Entsprechend habe ich jetzt nochmal darauf - mit bestehender Config - gedowngraded.

Sollte das Problem am Mittwoch wieder auftreten, werde ich einmal alles neu flashen und die "gute" Config zurückspielen.

Hast du eine Idee, wo ich die 3.53.30 von eQ3 finde? Dann hätte ich den Originalzustand aus November.

Wenn's dann immer noch nicht wieder dauerhaft läuft, bin ich wirklich am Ende meiner Ideen.

LG,
Marcel

**Reply by blackbasket** [reply]
Hi @jens-maus (und natürlich alle anderen),

ich habe letzte Woche alles abgelernt, die SD-Karte neu mit RM geflasht und alles neu gemacht.

Jetzt habe ich die gleichen Probleme schon wieder mit einer frischen Installation.

Aktuell habe ich nach 2 Tagen zwar noch HmIP-Kommunikation, dafür laufen keine Programme mehr und das WebUI hat sich auch verabschiedet. CPU und RAM sind lt SNMP unauffällig.

Ich verstehe es nicht - bitte helft mir :(

LG,
Marcel

**Reply by blackbasket** [reply]
Hier noch Logs, die ich herunterladen konnte... das letzte Programm wurde um 17:40 ausgeführt...
[messages_cut.txt](https://github.com/jens-maus/RaspberryMatic/files/6411856/messages_cut.txt)
[hmserver_cut.txt](https://github.com/jens-maus/RaspberryMatic/files/6411857/hmserver_cut.txt)
[lighttpd_cut.txt](https://github.com/jens-maus/RaspberryMatic/files/6411858/lighttpd_cut.txt)

**Reply by Bernd217** [reply]
Hallo Marcel, ich habe ein ähnliches Problem. Mein Webui ist jeden Freitag ab ca 3:00 und 6:00 Uhr plötzlich nicht mehr erreichbar. 
wie hast Du Dein Problem gelöst? Gruß Bernd
